swift/Tests/SwiftNoosphereTests/NoosphereTests.swift

```diff
@@ -8,6 +8,8 @@ final class NoosphereTests: XCTestCase {
         // reading from swift works as intended
         let noosphere = ns_initialize("/tmp/foo", "/tmp/bar", nil, nil)
 
+        ns_tracing_initialize(NS_NOOSPHERE_LOG_CHATTY.rawValue)
```

Comment:
:eyes: looks like it does format the enum values correctly! I'll need to look into https://github.com/subconsciousnetwork/noosphere/pull/340/files#diff-3359c09e1f3a2f311ceffbe341da1c49bf92026785af11d7bd1556aa28722d48R10 to see what's different here (u8 v u32?)

Reply:
Maybe an issue with a version of safer_ffi on a different machine? No idea why my other machine was rendering like that. Fixed in #340. Disregard!!